packages/api/lib/testUtils.js

```diff
@@ -95,6 +95,7 @@ function fakeGranuleFactory(status = 'completed') {
     cmrLink: 'example.com',
     productVolume: '100',
     duration: 0,
+    files: [],
```

Comment:
This is significant. `translatePostgresGranuleToApiGranule` (and therefor the apiGranule returned) now maintains an empty array instead of `undefined` if there are no files so the granule factory should reflect that.

Reply:
I'm backing out of this change in the latest commits. Updating the translate function to return an empty array instead of `undefined` when there are no files is perhaps the right approach but causes a slew of problems in reconciliation reports. We can revisit this conversation but I don't think we want to tackle that in the context of a bugfix that will be backported.